<launch>
  <arg name="simulation" default="false" />

  <node name="picking_object_manager" pkg="roomac_autonomous_manipulation" type="picking_object_manager.py" output="screen">
    <rosparam command="load" file="$(find roomac_autonomous_manipulation)/config/picking_object_manager_config.yaml" />

    <rosparam unless="$(arg simulation)" command="load" file="$(find roomac_autonomous_manipulation)/config/picking_object_manager_config_real.yaml" />
    <rosparam if="$(arg simulation)" command="load" file="$(find roomac_autonomous_manipulation)/config/picking_object_manager_config_simulation.yaml" />
  </node>
</launch>